test/abilities/disguise.test.ts

```diff
@@ -100,7 +100,7 @@ describe("Abilities - Disguise", () => {
     await game.phaseInterceptor.to("TurnEndPhase");
 
     expect(mimikyu.formIndex).toBe(disguisedForm);
-    expect(mimikyu.status?.effect).toBe(StatusEffect.POISON);
+    expect(mimikyu.getStatusEffect()).toBe(StatusEffect.POISON);
```

Comment:
```suggestion
    expect(mimikyu.getStatusEffect(true)).toBe(StatusEffect.POISON);
```
I wonder if the tests should be changed.